Source/Rows/TextAreaRow.swift

```diff
@@ -267,6 +273,16 @@ open class _TextAreaCell<T> : Cell<T>, UITextViewDelegate, AreaCell where T: Equ
             views["imageView"] = imageView
             dynamicConstraints.append(contentsOf: NSLayoutConstraint.constraints(withVisualFormat: "H:[imageView]-(15)-[textView]-|", options: [], metrics: nil, views: views))
             dynamicConstraints.append(contentsOf: NSLayoutConstraint.constraints(withVisualFormat: "H:[imageView]-(15)-[label]-|", options: [], metrics: nil, views: views))
+        } else if titlePercentage > 0.0 {
+            textView.textAlignment = .right
+            dynamicConstraints += NSLayoutConstraint.constraints(withVisualFormat: "H:[textView]-|", options: [], metrics: nil, views: views)
+            dynamicConstraints.append(NSLayoutConstraint(item: textView,
+                                                         attribute: .width,
+                                                         relatedBy: (row as? TextAreaConformance)?.titlePercentage != nil ? .equal : .lessThanOrEqual,
```

Comment:
You're right, will be concised to just titlePercentage in the next commit :)

Reply:
There are several problems with the situation of TextAreaRow. If I'm making any mistake in the research about how it works, please tell me, but here's what I reckon:

Consider the following code without my changes:
```
<<< TextAreaRow() {
                $0.title = "CHOCOLATE"
                $0.value = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor"
                $0.titlePercentage = 0.5
                $0.cell.imageView?.image = UIImage(named: "avatar.png")
                $0.textAreaHeight = .dynamic(initialTextViewHeight: 110)
            }

            <<< TextRow() {
                $0.title = "CHOCOLATE"
                $0.value = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor"
                $0.titlePercentage = 0.5
                $0.cell.imageView?.image = UIImage(named: "avatar.png")
            }
```

The vanilla result (except the yellow highlighting I use for debugging) is this:
<img width="384" alt="screenshot 2018-11-02 at 16 59 07" src="https://user-images.githubusercontent.com/25185916/47926468-e876ff80-dec0-11e8-9c76-4b9ffec4446a.png">

First off, the title does not appear, even without the picture. I assumed the titlePercentage addition to the TextAreaRow would make the text portion of the TextAreaRow aligned to single line rows with this property implemented, such as FieldRows. Second problem we have is with the imageView itself:

In the case of the FieldRow, the height of the cell is default, which is 44.0 or something. The code developed there does not even deal with constraints of the imageView, since the imageview comes from the "Cell" itself and it always sticks to the left, it's just stated that the label is to its right. The titlePercentage therefore considers only the space occupied by the text portion and the title portion.

With dynamically higher rows like TextAreaRow, the imageview scales, not only upwards but it also extends its width. What happens there is that it doesn't matter how we change the width of the textView to its right, the image scales to the height of the cell and does not care for any constraints unless we start meddling with turning off its "translatesAutoresizingMaskIntoConstraints" and determining its maximum size. That's why the imageView is so big. There's no alignment happening with other rows there if we have the imageView active, especially if the title does not appear at all.

The way I see it at least so far, this feature makes no sense with the imageView active, because no matter how many constraints we apply to the imageView, it won't budge, we can just determine the constraints of the textView. That's why I did it for the other case only.

Reply:
You are right that it doesn't make too much sense to support the case with an `imageView`.

Reply:
Can I only support the aforementioned case with the alignment with other FieldRows then?

Reply:
I have documented the variable as requested. Tried to use similar language as in the original titlePercentage. 

Without modifying the FieldRow or the TextAreaRow's original behavior, using this code:

`
let ipsum = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam"

        form +++
            Section(header: "Test of title percentage", footer: "")

            +++ Section("TextAreaRow")

            <<< TextAreaRow() {
                $0.value = ipsum
                $0.titlePercentage = 0.7
                $0.textAreaHeight = .dynamic(initialTextViewHeight: 110)
            }

            <<< TextRow() {
                $0.title = "CHOCOLATE"
                $0.value = ipsum
                $0.titlePercentage = 0.7
`

Changing the titlePercentage from 0.1 to 0.4 and 0.7 (tested all cases), we get the following behavior:

<img width="992" alt="titlepercentageshowcase" src="https://user-images.githubusercontent.com/25185916/48972409-d7399280-f029-11e8-84c0-2ebf2ed77427.png">